### ISSUE TITLE
Error with gallery block in block-field

### ISSUE THREAD

**Issue by skrichten** [NodeSquarespace/node-squarespace-server#140]
I have a block field in my site.region, and in the admin I put a gallery block in it.  When i load from the site from the local server, the gallery does not render and I see a js warning in my console.

```Returning, no wrapper found.```

Digging in Squarespace's code I see that this warning is output when it can't find an element with class ```sqs-gallery-container``` for the gallery.  Looking at the output of the local server, it looks like all the classes are put together with no space between them...

``` class="sqs-gallery-containersqs-gallery-block-gridsqs-gallery-aspect-ratio-squaresqs-gallery-thumbnails-per-row-1clear" ```

**Comment by kitajchuk** [comment]
That's super weird, I'll look into it.

**Comment by kitajchuk** [comment]
I just replicated this on a local copy of an un-modified `base-template`. It renders correctly the first time, but then fails when loading the block content from the local cache. Looks like that is where the issue is. Should be an easy fix :)

**Comment by kitajchuk** [comment]
I just figured out what it is. I have this `packStr` method in the server utilities for minifying html locally when caching. But, the issue is that Squarespace renders the className for that block with hard returns. The method splits on new lines and such to perform the string packing so it breaks in this case.

This is what the gallery block className looks like from Squarespace:

```html
<div class="
  sqs-gallery-container
  sqs-gallery-block-grid
  sqs-gallery-aspect-ratio-square
  sqs-gallery-thumbnails-per-row-1
  
  

  
  
  
  clear
" id="yui_3_17_2_1_1445707972133_129">
```

**Comment by kitajchuk** [comment]
Whoops, didn't mean to hit the "close and comment" button :P

**Comment by kitajchuk** [comment]
Pushed a fix for this one: deaa83f54e0a20faed20324701fa9882c67f85be

Published new server version: `+ node-squarespace-server@0.4.12`
